### PR TITLE
Add FirstBank

### PR DESCRIPTION
**Pull request by schwartzadev**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -305,6 +305,13 @@ websites:
       tfa: Yes
       sms: Yes
 
+    - name: First Bank (Colorado)
```
Comment (Jawshy) [inline _data/banking.yml]: Their website uses `FirstBank` without a space. Could you please amend your pull request to reflect that? Also, I am not sure if `(Colorado)` is necessary.

**Review by Jawshy — CHANGES_REQUESTED** [review]
Hi, @schwartzadev. Thanks for submitting a pull request. 

They list an email address on their [contact page](https://www.efirstbank.com/customer-service/contact-us.htm). Could you please add the following below line 311? 

```ruby
      email_address: banking@efirstbank.com
```

File: _data/banking.yml (hunk 1 above)
Comment (schwartzadev) [reply _data/banking.yml]: Sure thing

**Comment by schwartzadev** [comment]
Done. I also cross checked the case of FirstBank and it appears to be camel case on their website as well.

Also, there are multiple FirstBanks around and this one is based in CO so I think it is helpful to have the `(Colorado)`

**Comment by Carlgo11** [comment]
The logo is too big. Here's a compressed version:
1stbank

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo is too big. Here's a compressed version:
1stbank

**Review by Jawshy — APPROVED** [review]

File: _data/banking.yml [hunk 2]
```diff
@@ -330,6 +330,14 @@ websites:
       software: Yes
       doc: https://www.firsttechfed.com/OnlineServices/bankonlinesecurely/SecurityKey
 
+    - name: FirstBank (Colorado)
```
Comment (stephengroat) [inline _data/banking.yml]: I really think that the `(Colorado)` needs to be removed. From their [about page](https://www.efirstbank.com/about-firstbank/about.htm),

> FirstBank now has over $17 billion in assets and over 120 locations in **Colorado, Arizona, and California**

**Review by stephengroat — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 2 above)
Comment (schwartzadev) [reply _data/banking.yml]: On the contrary, there are multiple other FirstBanks, including [this](https://www.firstbanks.com/), [this](https://www.firstbank.com/), [this](http://www.1firstbank.com/vi/en), and [this](https://www.firstbankofberne.com/) one. I can take care of that as well.

Comment (stephengroat) [reply _data/banking.yml]: i think for how this site tends to be organized, those would be named `First Banks`, `First Bank & Trust Company`, `First Bank of Berne`, etc. those sites can be dealt with when submitted. thanks for the changes!

**Review by stephengroat — APPROVED** [review]

File: _data/banking.yml (hunk 2 above)
Comment (schwartzadev) [reply _data/banking.yml]: Okay that sounds good, thanks

**Comment by schwartzadev** [comment]
@Carlgo11 I think I resolved the logo issue in your request so this PR should be ready...

**Comment by schwartzadev** [comment]
@Carlgo11 I think I did your changes--please approve them for the merge

**Review by Carlgo11 — APPROVED** [review]

**Review by jamcat22 — APPROVED** [review]

**Comment by jamcat22** [comment]
Thanks for the PR @schwartzadev! You can learn more about our merging process on our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Merge) page.

**Pull request merged**